modules/jpg/register_types.cpp

```diff
@@ -32,14 +32,14 @@
 
 #include "image_loader_jpegd.h"
 
-static ImageLoaderJPG *image_loader_jpg = nullptr;
+static Ref<ImageLoaderJPG> image_loader_jpg;
 
 void initialize_jpg_module(ModuleInitializationLevel p_level) {
 	if (p_level != MODULE_INITIALIZATION_LEVEL_SCENE) {
 		return;
 	}
 
-	image_loader_jpg = memnew(ImageLoaderJPG);
+	image_loader_jpg.instantiate();
```

Comment:
We seem to be flipping between memnew and instantiate. What's the difference?

Reply:
memnew calls `new Obj`.
`instantiate` is a function of `RefCounted`.
`ref_counted = memnew(Object)` is not valid